src/main.js

```diff
@@ -189,6 +189,11 @@ app.on('ready', function() {
         title: arg.title,
         content: arg.options.body
       });
+
+      /* Todo: add idle here */
```

Comment:
"add idle here" might make confusing later. Could you change the comment? for example, "add detection for idle state".